docs-dev/packages.md

```diff
@@ -16,14 +16,22 @@ This repository contains three services - `aziot-certd`, `aziot-identityd` and `
 <td><code>centos:7</code></td>
 </tr>
 <tr>
-<td>Debian 9 / Raspbian 9</td>
+<td>RHEL 8 compatible</td>
+<td><code>almalinux:8.4</code></td>
```

Comment:
Is there a reason to not use `centos:8` ?

Reply:
Only b/c CentOS 8 hits EOL in a few months so I figured it would be better to use something expected to have a longer lifetime.

Reply:
Hmm, I thought they'd update the tag for Stream, but it looks like they publish a separate `stream8` tag and that too only on quay.io ( https://wiki.centos.org/FAQ/CentOSStream#What_artifacts_are_built.3F -> https://quay.io/repository/centos/centos?tab=tags vs https://hub.docker.com/_/centos?tab=tags&page=1&ordering=last_updated)

Also, I guess for the periods of time where Stream is ahead of RHEL we could have a situation where we link to newer libs / symbols than RHEL and so our packages become uninstallable / unrunnable.

This is okay then.

Do you have any reason to pick AlmaLinux over RockyLinux? (IIRC we're even sponsoring that one.)

Reply:
> Do you have any reason to pick AlmaLinux over RockyLinux? (IIRC we're even sponsoring that one.)

No. But, now that I've learned of [RedHat's UBI images](https://developers.redhat.com/products/rhel/ubi) I'm recommending one of those instead (at least for the container to use in building). 

Note this PR still suggests using an AlmaLinux VM for the tests - only b/c it doesn't incur any additional cost on top of Azure infrastructure costs.  There isn't an equivalent "free" one for Rocky Linux.

Fwiw, I believe Microsoft is doing some sponsoring of both projects.